demo/drag-drop-dataTransfer.ts

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
thanks for putting an example, but user can't simply run this in a browser to test/play with it. do you mind putting a generic JS examples like the others ?
(I happen to use Angular myself but most don't as this lib isn't tied to any framework). thanks!

Reply:
Hi, I have just included a generic JS example.